src/persist/src/indexed/encoding.rs

```diff
@@ -164,6 +193,16 @@ pub struct BlobTraceBatchMeta {
     pub level: u64,
 }
 
+impl PartialEq for BlobTraceBatchMeta {
+    fn eq(&self, other: &Self) -> bool {
+        self.key.eq(&other.key)
+            && self.desc.upper()[0].eq(&other.desc.upper()[0])
```

Comment:
I don't love this -- I wonder if theres a more idiomatic way to derive PartialEq for Antichains when the timestamp type is ... something

Reply:
PartialEq is already derived for Antichain<T> where T: PartialEq. Am I misunderstanding?

https://docs.rs/timely/0.12.0/timely/progress/frontier/struct.Antichain.html#impl-PartialEq%3CAntichain%3CT%3E%3E

Reply:
Oh! I see, the issue isn't Antichain, it's Description. You can make a newtype wrapper around Description that impls PartialEq and Derefs to Description (the Deref allows it to transparently act as a Description):

```
struct<T> Foo<T>(Description<T>);

impl<T: PartialEq> PartialEq for Foo { ... }
impl<T: Eq> Eq for Foo { ... }
impl Deref for Foo {
   type Target = Description<T>;
}

Reply:
I spoke to Frank -- he's happy to have PartialEq and Eq for Description when the underlying timestamp type implements those traits. I'll go ahead and PR that up, and then we can just derive those traits for everything here once that lands